app/controllers/users_controller.rb

```diff
@@ -59,7 +57,11 @@ def destroy
 
   private
 
-  def user_params
+  def set_user
+    @user = User.find(params[:id])
+  end
+
+  def allowed_user_params
     if current_user.admin?
```

Comment:
Done. %i was new to me, but now that I read about it, this is a very good suggestion.